docs/modules/ROOT/pages/cache-configuration.adoc

```diff
@@ -1,59 +1,65 @@
-= Configuring Cache
-:description: pass:q[Hazelcast offers a specification-compliant JCache implementation. The Hazelcast JCache implementation is 100% TCK (Technology Compatibility Kit) compliant and therefore passes all specification requirements. You can create and configure JCache using the `Cache` custom resource.]
+= Configure Cache
+:description: pass:q[Hazelcast offers a specification-compliant JCache implementation. The Hazelcast JCache implementation is 100% Technology Compatibility Kit (TCK) compliant and therefore passes all specification requirements. You can create and configure JCache using the Cache custom resource.]
 
 {description}
 
-== Configuring a Cache Resource
+== Configure a Cache Resource
 
-You can use any of the following configuration options in the `Cache` resource. For more information, see the xref:api-ref.adoc#cachespec[API Reference].
+The Cache resource is configured using the following fields: 
```

Comment:
I wasn't sure if a resource and a custom resource were the same or different, but having followed the instructions I can see all are CRs - update "resource" to "custom resource" throughout.

Side comment; for search indexing do we need to include K8s and CR as well?

Reply:
>I wasn't sure if a resource and a custom resource were the same or different, but having followed the instructions I can see all are CRs - update "resource" to "custom resource" throughout.

most of the time, resource = custom resource in our doc but sometime we mean k8s resources like pod, job, statefulset when we say resource but you can easily understand from the context.

 >Side comment; for search indexing do we need to include K8s and CR as well?
 
 like Kubernetes Custom Resource?

Reply:
For consistency, I'd prefer to use a single term; saves anyone wondering whether we mean the same thing or not.

Yes, like Kubernetes (K8s) and custom resource (CR); it would just be at first use on the page - and is only to help with searches.